src/components/Header.js

```diff
@@ -1,9 +1,32 @@
-import React, { Component } from "react";
+import React, { Component } from 'react';
+import { NavLink } from 'react-router-dom';
+import '../App.css'
 
 class Header extends Component {
   render() {
-    return <h1>Header is right here</h1>;
+    return(
+        <>
+          <header>
+            <section className='header'>
+                <NavLink to = "/" >
+                </NavLink>
+            </section>  
+
+              <section className='navbar'>
+                 <div className='nav-links'>
+                   <ul className="navbar-nav me-auto order-0">
+                     <NavLink to = "/catindex" className='navitem'>We are all cats</NavLink>  
+                   </ul>
+                   <ul className="navbar-nav me-auto order-0"> 
+                     <NavLink to = "/catnew" className='navitem'>Cats are fun</NavLink>
```

Comment:
Be proactive about keeping indentation clean. It is best practice to have no spaces in the attributes inside your tags like this `className='navbar'`